install_ffmpeg.sh

```diff
@@ -35,7 +36,7 @@ fi
 
 if [ ! -e "$HOME/nasm/nasm" ]; then
   # sudo apt-get -y install asciidoc xmlto # this fails :(
-  git clone -b nasm-2.14.02 https://repo.or.cz/nasm.git "$HOME/nasm"
+  git clone -b nasm-2.14.02 https://github.com/iameli/nasm.git "$HOME/nasm"
```

Comment:
Is this needed?

Reply:
Oops, no - `repo.or.cz` was briefly down while I was hacking on this. Switched back.